routes/v3/apps.ts

```diff
@@ -11,14 +11,14 @@ const router = new Router({
 
 router.use(errorHandler);
 
-router.get('/', auth.jwt, async (ctx, next) => {
-  /*const query = {
-    installed: ctx.request.query.installed === '1',
-  };
-  const apps = await appsLogic.get(query);
-  ctx.body = apps;*/
+router.get('/', auth.jwt, async (_ctx, _next) => {
+  // Const query = {
+  //   installed: ctx.request.query.installed === '1',
+  // };
+  // const apps = await appsLogic.get(query);
+  // ctx.body = apps;
+  // await next();
   return [];
```

Comment:
```suggestion
  ctx.body = [];
  await next();
```

This is what should actually be here